sites/fast-website/src/app/components/fast-frame/fast-frame.ts

```diff
@@ -161,12 +168,21 @@ export class FastFrame extends FASTElement {
         this.backgroundColor = this.previewBackgroundPalette[this.lastSelectedIndex];
     };
 
+    private resizeHandler = (): void => {
+        // Reset expanded when screensize is greater than 660
+        if (window.innerWidth > 660) {
+            this.expanded = false;
+        }
+    };
+
     constructor() {
         super();
 
         const accentColorHSL = rgbToHSL(parseColorHexRGB(this.accentColor)!);
         this.hue = accentColorHSL.h;
         this.saturation = accentColorHSL.s;
         this.lightness = accentColorHSL.l;
+
+        window.addEventListener("resize", this.resizeHandler, { passive: true });
```

Comment:
I'd recommend using [window.matchMedia](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia) here instead of a resize listener. It is more performant and should result in a smoother resize.